core/src/display_object/movie_clip.rs

```diff
@@ -4554,7 +4557,9 @@ impl<'gc, 'a> MovieClip<'gc> {
                             .library
                             .library_for_movie_mut(movie.clone());
 
-                        match library.character_by_id(id) {
+                        let character = library.character_by_id(id);
```

Comment:
This needs to be duplicated (here and in `lookup_class_for_character`) to satisfy the borrow checker

Reply:
I'm guessing `Character` should just become a `Copy`, if/when all its variants become `Gc` wrappers - that should solve it, right?